service/src/main/java/bio/terra/workspace/amalgam/landingzone/azure/LandingZoneApiDispatch.java

```diff
@@ -192,6 +199,23 @@ public Optional<ApiAzureLandingZoneDeployedResource> getSharedBatchAccount(
     return getSharedResourceByType(bearerToken, landingZoneId, AZURE_BATCH_ACCOUNT_RESOURCE_TYPE);
   }
 
+  public Optional<ApiAzureLandingZoneDeployedResource> getSharedKubernetesCluster(
+      BearerToken bearerToken, UUID landingZoneId) {
+    return getSharedResourceByType(
+        bearerToken, landingZoneId, AZURE_KUBERNETES_CLUSTER_RESOURCE_TYPE);
+  }
+
+  public Optional<ApiAzureLandingZoneDeployedResource> getSharedDatabase(
```

Comment:
Was just wondering -- did we bump the LZ definition version with the change to postgres flex? Just anticipating Leo needing to make some decisions at app deployment time:
1. if this is a v2 Landing Zone, do all the new stuff (create WSM database, identity, etc)
2. otherwise, current behavior

Reply:
I did not bump the LZ version